catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistrySubscriptions.java

```diff
@@ -114,13 +114,18 @@ private void addOrUpdateMetacardsForRemoteRegistries(Set<String> sourceIds)
     private Map<String, Metacard> getRegistryMetacardsMap() throws FederationAdminException {
 
         Map<String, Metacard> registryMetacardsMap = new HashMap<>();
-        List<Metacard> registryMetacards = federationAdminService.getRegistryMetacards();
+        try {
+            List<Metacard> registryMetacards = javax.security.auth.Subject.doAs(subject,
```

Comment:
The system calls this method through a camel route and it isn't available from an endpoint. I also am pretty sure there are other methods in this class getting privileges in the same manor. 

If this needs to be changed, we should probably re-evaluate the other methods in this class as well.